contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -335,23 +336,4 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
         if (CUSTOM_GAS_TOKEN.balanceOf(address(this)) < requiredL1CallValue) revert InsufficientCustomGasToken();
         return requiredL1CallValue;
     }
-
-    function _from18ToNativeDecimals(uint256 amount) internal view returns (uint256) {
```

Comment:
I think we can keep this if we assume that all legacy inboxes, like the one we call with AlephZero, use 18 decimal precision

Reply:
And then we pass in the precision number

Reply:
Sounds good https://github.com/across-protocol/contracts/pull/699/commits/6437f7d55cc9c274576350617b28cbb08bb65c10